lib/firespring_dev_commands/templates/aws/services/route53.rb

```diff
@@ -0,0 +1,109 @@
+require_relative '../../base_interface'
+
+module Dev
+  module Template
+    class Aws
+      module Services
+        # Class contains rake templates for managing your AWS settings and logging in
+        class Route53 < Dev::Template::BaseInterface
+          # Create the rake task which ensures active credentials are present
+          def create_ensure_credentials_task!
+            # Have to set a local variable to be accessible inside of the instance_eval block
+            exclude = @exclude
+
+            DEV_COMMANDS_TOP_LEVEL.instance_eval do
+              return if exclude.include?(:ensure_aws_credentials)
+
+              task ensure_aws_credentials: %w(init) do
+                raise 'AWS Credentials not found / expired' unless Dev::Aws::Credentials.new.active?
+              end
+            end
+          end
+
+          # Create the rake task for the hosted zone method
+          def create_dns_logging_activate_task!
+            # Have to set a local variable to be accessible inside of the instance_eval block
+            exclude = @exclude
+
+            DEV_COMMANDS_TOP_LEVEL.instance_eval do
+              namespace :aws do
+                return if exclude.include?(:dns_logging)
+
+                namespace :hosted_zone do
+                  namespace :dns_logging do
+                    desc 'Activates query logging for all hosted zones by default.' \
+                         'This command should be run from the account the hosted zone(s) reside.' \
+                         "\n\t(Required) Specify HOSTED_ZONE_GROUP='arn:aws:logs:REGION:ACCOUNT_ID:' to specify the ARN of the target log group." \
```

Comment:
Can we rename this "LOG_GROUP_ARN" or something more descriptive? Also, can we default the "arn" piece of it? We should just need to pass in the group name.